packages/app/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@aerogear/app",
+  "version": "0.2.1",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "clean": "del coverage src/**/*.js src/**/*.map test/**/*.js test/**/*.map",
+    "build": "tsc",
+    "watch": "tsc --watch",
+    "test": "nyc mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aerogear/aerogear-js-sdk.git"
+  },
+  "keywords": [
+    "aerogear",
+    "aerogear services"
+  ],
+  "author": "AeroGear",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/aerogear/aerogear-js-sdk/issues"
+  },
+  "homepage": "https://github.com/aerogear/aerogear-js-sdk#readme",
+  "devDependencies": {
+    "mocha": "^5.1.1",
+    "@types/chai": "^4.1.3",
+    "chai": "^4.1.2",
+    "nyc": "^11.7.1"
+  },
+  "dependencies": {
+    "@aerogear/core": "0.2.1"
```

Comment:
If we have core as dependency, and app will be always installed, could we have core as "peerDependency" in modules? (auth, push, etc)

Reply:
Comment of the year! I'm going to check that!

Reply:
Done.